build.sbt

```diff
@@ -7,8 +8,10 @@ lazy val root = (project in file("."))
     scalaVersion := "2.13.1",
     crossScalaVersions := Seq(scalaVersion.value, "2.12.10"),
     libraryDependencies ++= Seq(
-      "is.cir" %% "ciris" % "1.2.1",
-      "software.amazon.awssdk" % "ssm" % "2.16.48"
+      "is.cir" %% "ciris" % "2.0.1",
+      "software.amazon.awssdk" % "ssm" % "2.16.78",
+      "org.typelevel" %% "cats-effect-kernel" % catsEffectVersion,
```

Comment:
Do we definitely need to specify the these cats effect versions ? Is that not included as part of is.ciris ?

Reply:
cats-effect-kernel is brought in with ciris, but cats-effect is not. (CE3 seems to have restructured into a few different packages.) I'll remove cats-effect-kernel from the dependencies though :+1:

Reply:
Thanks, will approve